### DISCUSSION TITLE
How to update Raspiblitz on Proxmox VM

### DISCUSSION THREAD

**Discussion by ciori** [raspiblitz/raspiblitz#4206] [Q&A]
Hi, what is the preferred procedure to update a Raspiblitz instance that was installed following the Proxmox VM guide?

Do you just need to run the "build_sdcard.sh" script again or are there some other manual steps to do?

**Comment by adevoss** [comment]
What I do is:
- fresh install Debian (11 for v1.10)
- `apt install sudo wget`
- download and execute as root `build_sdcard.sh` with options

**Reply by ciori** [reply]
Ok, so you basically replace the VM with a new one mounting the same data disk?
Isn't there a less "disruptive" way to just upgrade the same system?

**Comment by adevoss** [comment]
> replace the VM

Yes. This is the upgrade method of Raspiblitz (replace SD card). SD card failures on Raspberry Pi are common. Maybe I should say: not if but when.
> less "disruptive" way

Yes. There is an upgrade option in the menu BUT then you're on your own. It's a developer option I think.

**Comment by alex27riva** [comment]
Maybe is possible to use a Proxmox template VM to save time?
